services/121-service/src/transaction-queues/dto/onafriq-transaction-job.dto.ts

```diff
@@ -1,10 +1,8 @@
-export interface OnafriqTransactionJobDto {
-  readonly programId: number;
-  readonly programFspConfigurationId: number;
-  readonly paymentNumber: number;
+import { SharedTransactionJobDto } from '@121-service/src/transaction-queues/dto/shared-transaction-job.dto';
+
+export interface OnafriqTransactionJobDto extends SharedTransactionJobDto {
   readonly referenceId: string;
   readonly transactionAmount: number;
-  readonly isRetry: boolean;
   readonly userId: number;
```

Comment:
- userId is now both in shared and in this specific one.
- Also, I guess referenceId/transactionAmount/bulkSize can also move to shared (from all specific ones).